openapi/p4pa-debt-position.openapi.yaml

```
@@ -0,0 +1,273 @@
+openapi: 3.0.1
+info:
+  title: p4pa-debt-positions
+  description: Api and Models
+  version: 0.0.1
+servers:
+  - url: http://localhost:8080
+    description: Generated server url
+paths:
+  /api/debt-positions:
+    post:
+      tags:
+        - debt-position-controller
```

I would avoid the "controller" name in tags ;)
```suggestion
        - debt-position
```